pom.xml

```diff
@@ -2484,7 +2484,10 @@
                 <configuration>
                     <properties>
                         <configurationParameters>junit.jupiter.execution.timeout.thread.mode.default = SEPARATE_THREAD
-                            junit.jupiter.extensions.autodetection.enabled = true</configurationParameters>
+                            junit.jupiter.extensions.autodetection.enabled = true
+                            junit.jupiter.execution.parallel.enabled = true
+                            junit.jupiter.execution.parallel.mode.default = concurrent
```

Comment:
any reason not to take `air.test.thread-count` into account?
is it going to be deprecated?

(this would be sth like
```
junit.jupiter.execution.parallel.config.strategy=fixed
junit.jupiter.execution.parallel.config.fixed.parallelism=${air.test.thread-count}
```
)

Reply:
See https://github.com/trinodb/trino/pull/19554#discussion_r1375309502